src/main/kotlin/com/hibob/academy/dao/OwnerDao.kt

```diff
@@ -0,0 +1,48 @@
+package com.hibob.academy.dao
+
+import jakarta.inject.Inject
+import org.jooq.DSLContext
+import org.jooq.Record
+import org.jooq.RecordMapper
+import java.util.*
+import kotlin.random.Random
+
+class OwnerDao @Inject constructor(private val sql: DSLContext) {
+
+    private val owner = OwnerTable.instance
+    private val pet = PetTable.instance
+
+    private val ownerMapper = RecordMapper<Record, OwnerData> { record ->
+        OwnerData(
+            record[owner.id],
+            record[owner.name],
+            record[owner.companyId],
+            record[owner.employeeId]
+        )
+    }
+
+    fun getOwner(companyId: Long): List<OwnerData> =
+        sql.select(owner.name, owner.companyId, owner.employeeId)
+            .from(owner)
+            .where(owner.companyId.eq(companyId))
+            .fetch(ownerMapper)
+
+    fun createOwner(id: Long, ownerName: String, companyId: Long, employeeId: String) =
+        sql.insertInto(owner)
+            .set(owner.id, id)
```

Comment:
Id is autogenerated when creating a inserting a new row, no need to add it here.